lib/cypress/e2e/index.cy.js

```diff
@@ -31,6 +31,10 @@ describe("@storyblok/astro", () => {
     cy.visit("http://localhost:3000/");
     cy.get("[data-test=custom-fallback-component]").should("exist");
   });
+  it("Richtext Renderer renders embedded bloks correctly", () => {
```

Comment:
Love it! Simple yet powerful test

Reply:
Only following @manuelschroederdev's lead! 😄